src/main/kotlin/com/terraformation/backend/nursery/db/BatchStore.kt

```diff
@@ -731,4 +782,41 @@ class BatchStore(
           .execute()
     }
   }
+
+  private fun fetchSubLocationIds(batchId: BatchId): Set<SubLocationId> {
+    return dslContext
+        .select(BATCH_SUB_LOCATIONS.SUB_LOCATION_ID)
+        .from(BATCH_SUB_LOCATIONS)
+        .where(BATCH_SUB_LOCATIONS.BATCH_ID.eq(batchId))
+        .fetchSet(BATCH_SUB_LOCATIONS.SUB_LOCATION_ID.asNonNullable())
+  }
+
+  private fun updateSubLocations(
+      batchId: BatchId,
+      facilityId: FacilityId,
+      existingSubLocationIds: Set<SubLocationId>,
+      desiredSubLocationIds: Set<SubLocationId>
+  ) {
+    //    val existingSubLocationIds = fetchSubLocationIds(batchId)
```

Comment:
Delete comment?

Reply:
Sigh, managed to forget to push this to the PR branch before merging it, but good catch and I will remove this in a separate PR.